scripts/getBuildData.js

```diff
@@ -0,0 +1,26 @@
+const git = require('git-rev-sync'); /* eslint-disable-line import/no-extraneous-dependencies */
+
+module.exports = (isCompiled = false) => {
+  let buildData = '';
+
+  if (git.branch() !== 'master') {
+    buildData += '\n';
+
+    if (isCompiled) {
+      buildData += `
+        Build Date: ${(new Date()).toISOString()}
+        Node Version: ${process.version}`;
+    }
+
+    buildData += `
+      Branch: ${git.branch()}
+      Commit: ${git.long()}`;
+
+    if (process.env.CIRCLECI) {
+      buildData += `
+        Build Number (CircleCI): ${process.env.CIRCLE_BUILD_NUM}`;
+    }
+  }
+
+  return buildData.replace(/\n/, '').replace(/^  +/gm, '  ');
```

Comment:
That's totally fair and up to you, not really important. Personally I've always been kinda against multiline strings like this, especially where whitespace within the string matters. I feel that whitespace within (most) code should be for code structure like indenting and keeping things organized visually. I don't really like when an actual value is effected by this because it's caused me trouble in the past. But again, it's a super minor thing so I agree with not changing it.